packages/manager/modules/pci/src/projects/project/storages/containers/list/list.routing.js

```diff
@@ -0,0 +1,64 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.storages.objects.list', {
+    url: '/list',
```

Comment:
We should avoid `/list` in urls, if we want to access an object list it should be under `/objects` then for a detail or something else `/objects/:id` or `/objects/something`

Reply:
Hmm i already have the overall containers for /objects. Ill rename these things. Its a bit confusing

Reply:
@varun257 `/objects` was taken as an example but basically we can replace with `/<productName>`

Reply:
I will work on this once other PRs are merged. Since this PR has refactoring of the code and names, it would be easier to work in the end.